variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/factories/VariantVcfFactory.java

```diff
@@ -351,4 +361,38 @@ protected static int mapToMultiallelicIndex(int parsedAllele, int numAllele) {
         }
         return correctedAllele;
     }
+
+    private void checkVariantInformation(Variant variant, String fileId,
+                                           String studyId) throws NonVariantException, IncompleteInformationException {
```

Comment:
Fix indentation, and the last parameter can still fit in the first line.

Reply:
👍